daemon/src/lib.rs

```diff
@@ -46,16 +47,20 @@ pub mod wallet;
 pub mod wallet_sync;
 pub mod wire;
 
-const HEARTBEAT_INTERVAL: std::time::Duration = Duration::from_secs(5);
+pub const HEARTBEAT_INTERVAL: std::time::Duration = Duration::from_secs(5);
 
 pub const N_PAYOUTS: usize = 200;
 
+/// Struct controlling the lifetime of the actors
```

Comment:
I think this needs more docs. Can you explain a bit here, how this works? :)

Reply:
Also, I think `Tasks` might be a better name because it is not just actors but also notifier tasks.

Reply:
sure! `Tasks` sounds like good name. I'll describe more works.

> Nice work!
> 
> A couple of changes seem to be unrelated, would you mind getting rid of those again? Also, if you really want to make the heartbeat interval configurable, can we do that as a separate patch?



> Nice work!
> 
> A couple of changes seem to be unrelated, would you mind getting rid of those again? Also, if you really want to make the heartbeat interval configurable, can we do that as a separate patch?

Sure, I'll remove heartbeat interval changes from this one (it's not that related, it was only a test optimisation, and can go in later / if at all)